vgen/src/test/java/io/gapi/vgen/testdata/go_library.baseline

```diff
@@ -75,6 +76,22 @@ var (
 	bookPathTemplate = gax.MustCompilePathTemplate("shelves/{shelf}/books/{book}")
 )
 
+func idempotentRetryCodes() []gax.CallOption {
+	return []gax.CallOption{
+		gax.WithRetryCodes([]codes.Code{
+			codes.DeadlineExceeded,
```

Comment:
Since these codes are only used on idempotent functions, I believe there should be no issue even if it is a RPC time out